MDLParser.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
-#include <cstdint>
+#include <stddef.h>
+#include <stdint.h>
```

Comment:
Could you use `using` instead to put the appropriate types from cstdint into the global namespace? (And same for size_t in cstddef)

I doubt I’ve been consistent with c* vs *.h headers in this (or other) projects, but I’ve since learned that *.h STL headers are dep

Reply:
I was doing the quick and lazy fix, the (u)int*_t types are used literally everywhere without std, but sure

Reply:
That looks a lot better you're right